doc/question.md

```diff
@@ -97,14 +101,29 @@ The `question.html` is a template used to render the question to the student. A
 <p>$F = $ <pl-number-input answers_name="F" comparison="sigfig" digits="2" /> $\rm m/s^2$
 ```
 
-The `question.html` is regular HTML, with three special features:
+The `question.html` is regular HTML, with four special features:
 
 1. Any text in double-curly-braces (like `{{params.m}}`) is substituted with variable values. This is using [Mustache](https://mustache.github.io/mustache.5.html) templating.
 
 2. Special HTML elements (like `<pl-number-input>`) enable input and formatted output. See the [list of PrairieLearn elements](elements.md).
    
 3. A special `<markdown>` tag allows you to write Markdown inline in questions.
 
+4. LaTeX equations are available by putting `$y = x^2$` or `\[ y = \int_0^1 x^2\ dx \]` into the HTML.
```

Comment:
How about a distinction between `inline` and `display`?

```
Inline: `$y = x^2$` or `\(a^2+ b^2 = c^2\)`
Display: 
`$$\phi = \frac{1 + \sqrt{5}}{2}$$`
`\[ y = \int_0^1 x^2\ dx \]`

Reply:
done